app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryStateActions.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CollapsedStoryPoints from './CollapsedStoryPoints';
+import CollapsedStoryStateButton from './CollapsedStoryStateButton';
+import { isStoryNotEstimated } from '../../../rules/story';
```

Comment:
We could move it to `app/assets/javascripts/models/beta/story.js` as well since it's a domain rule.